app/vstda.ctrl.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('myApp')
+        .controller('MainCtrl', MainCtrl);
+
+    MainCtrl.$inject = ['$scope', '$filter'];
+
+    /* @ngInject */
+    function MainCtrl($scope, $filter) {
+      var vm=this;
+      $scope.newItem = [];
```

Comment:
You should refactor this to use the vm instead of $Scope.  That is why the assignment is made on line 12.